mne/commands/mne_watershed_bem.py

```diff
@@ -0,0 +1,166 @@
+"""
+
+    Create BEM surfaces using the watershed algorithm included with
+        FreeSurfer
+
+"""
+
+from __future__ import print_function
+import sys
+import os
+import shutil
+import mne
+from mne.utils import (verbose, logger)
+
+
+@verbose
+def mne_watershed_bem(subject=None, subjects_dir=None, overwrite=False,
+                      volume='T1', atlas=False, gcaatlas=False, preflood=None,
+                      verbose=None):
+    """
+    Create BEM surfaces using the watershed algorithm included with FreeSurfer
+
+    Parameters
+    ----------
+    subject : string
+        Subject name (SUBJECT environment variable)
+    subjects_dir : string
+        Directory containing subjects data. If None use the Freesurfer\
+         SUBJECTS_DIR environment variable.
+    overwrite : bool
+        Write over existing files
+    volume : string
+        Defaults to T1
+    atlas : bool
+        Specify the --atlas option for mri_watershed
+    gcaatlas : bool
+        Use the subcortical atlas
+    preflood : int
+        Change the preflood height
+    verbose : bool, str or None
+        If not None, override default verbose level
+    """
+    if not os.environ['FREESURFER_HOME']:
+        raise RuntimeError('FREESURFER_HOME environment variable not set')
+    if subject:
+        os.environ['SUBJECT'] = subject
+    else:
+        try:
+            subject = os.environ['SUBJECT']
+        except KeyError:
+            raise RuntimeError('SUBJECT environment variable not defined')
+    if subjects_dir:
+        os.environ['SUBJECTS_DIR'] = subjects_dir
+    else:
+        try:
+            subjects_dir = os.environ['SUBJECTS_DIR']
+        except KeyError:
+            raise RuntimeError('SUBJECTS_DIR environment variable not defined')
```

Comment:
We have a standard way of getting this through `get_subjects_dir`, see other uses throughout the code base. It should be something like `subjects_dir = get_subjects_dir(subjects_dir, raise_error=True)`